packages/stentor-utils/src/date-time/isISO8601Duration.ts

```diff
@@ -0,0 +1,28 @@
+/*! Copyright (c) 2020, XAPPmedia */
+
+/**
+ * Regex to detect ISO-8601 duration strings
+ * 
+ * {@link https://stackoverflow.com/a/32045167}
+ */
+export const ISO_8601_DURATION = /^P(?!$)(\d+(?:\.\d+)?Y)?(\d+(?:\.\d+)?M)?(\d+(?:\.\d+)?W)?(\d+(?:\.\d+)?D)?(T(?=\d)(\d+(?:\.\d+)?H)?(\d+(?:\.\d+)?M)?(\d+(?:\.\d+)?S)?)?$/;
```

Comment:
Is this the same regex as the "const splitter" attribute in the `getDurationForm.ts` file?

Reply:
Not exactly, one (this one) is for detecting validity of a ISO-8601 duration string and the other is for pulling out the parts of a ISO-8601 duration string.  I tried using the part puller for the detection one and a test failed, since I did not author these I am hesitant to change them.